### PR TITLE
Fix grammar on site

### PR DESCRIPTION
**Pull request by stevepiercy**

### CODE REVIEW

File: src/lib/PackageList.svelte [hunk 1]
```diff
@@ -4,7 +4,7 @@
   import PackageItem from "$lib/PackageItem.svelte";
 </script>
 
-<div class="results_count">We found: {$results_count} Plone add-on's</div>
+<div class="results_count">We found: {$results_count} Plone packages</div>
```
Comment (MrTango) [inline src/lib/PackageList.svelte]: I'm sure we should change the naming here. So far in the Plone world we always called them add-on and not package. Hence the name of the project.

Comment (stevepiercy) [reply src/lib/PackageList.svelte]: The terms are not used consistently. In the UI you refer to packages in two different places, and add-on is a type of package.

Screenshot 2024-10-25 at 2 57 58 AM

Also I think "Plone Add-on Gallery" is misnamed and should be "Plone Package Gallery", based on what it actually searches for.

Comment (MrTango) [reply src/lib/PackageList.svelte]: Then i think i rather change the Package Types in to "Addon Types", as everywhere in Plone we call them addons.
I know the Python world calls them differently but that is a different context. In Plone extensions are called addons, even in Volto that is the case.

Comment (stevepiercy) [reply src/lib/PackageList.svelte]: Well, npm calls them "packages", so it's not just Python.

I'll replace "package" with "add-on", though, just for consistency. Inconsistency bugs me more than exact name.

Comment (stevepiercy) [reply src/lib/PackageList.svelte]: @MrTango I think I got it now. Please check. Thank you!

**Review by MrTango — APPROVED** [review]
thx

**Pull request merged**